index.js

```diff
@@ -3,7 +3,6 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+/* eslint-disable no-unused-vars*/
 const nodeMajorVersion = +process.versions.node.split('.')[0];
-module.exports = nodeMajorVersion >= 7 ?
-  require('./dist/src') :
-  require('./dist6/src');
+module.exports = require('./dist/src');
```

Comment:
Can we switch this to be just `./dist` like we have in `loopback-next` packages?

Reply:
Yes, good catch :-).